src/renderer/renderElement.ts

```diff
@@ -14,8 +14,10 @@ import { Drawable, Options } from "roughjs/bin/core";
 import { RoughSVG } from "roughjs/bin/svg";
 import { RoughGenerator } from "roughjs/bin/generator";
 import { SceneState } from "../scene/types";
-import { SVG_NS, distance } from "../utils";
+import { SVG_NS, distance, getPixelRatio } from "../utils";
```

Comment:
As a follow up, we should update all the places where we read window.devicePixelRatio to use it, I believe there are more in the codebase.